subsystems/SolenoidSubsystem.java

```diff
@@ -3,7 +3,7 @@
 
 import org.usfirst.frc4904.standard.commands.solenoid.SolenoidSet;
 import edu.wpi.first.wpilibj.DoubleSolenoid;
-import edu.wpi.first.wpilibj.command.Subsystem;
+import edu.wpi.first.wpilibj2.command.Subsystem;
```

Comment:
this file is nowhere near complete but it is beyond the scope of chassis fixes so don't worry about it